operator/gitops/argocd/grafana/dashboards/pipeline-service-dashboard.json

```diff
@@ -87,14 +87,11 @@
         "y": 0
       },
       "id": 93,
+      "repeat": "datasource",
```

Comment:
rather than those datasource ... ${DS_PROMETHEUS-APPSTUDIO-DS} entries, I saw this used in the rows/panels that "rendored immediately" and did not need me to initiate a manual "perform query" to fill out the alert panels.

Using this, I saw the alert panels rendor once the 30 minute window for our sample range passed in my dev_setup env

in stage/prod, since they are long running, we typically should have those metrics available from 30 minute ago and see as soon as we log in